configuration.nix

```diff
@@ -0,0 +1,53 @@
+{ config, pkgs, lib, ... }:
+
+{
+
+  nixpkgs.config =
+    {
+      allowUnfree = true;
+    };
+
+  networking.firewall = {
+    enable = true;
+    allowedTCPPorts = [ 22 443 19999 3000 ];
+  };
+
+  services.openssh = {
+    enable = true;
+    passwordAuthentication = false;
+
+  };
+
+  users.users."root".openssh.authorizedKeys.keys = [
+    "ssh-ed25519 AAAACNzaC1lZDI1NTE5AAAAIPwjDcLZ5jrK3XL5dF7eVb+9wPWqUBLf4NCuRzmRiteT steven@DESKTOP-4IURNHR"
+  ];
+
+  services.netdata = {
+    enable = true;
+    user = "root";
```

Comment:
Why must netdata be run as root?

Reply:
It is a good point I will leave it as the default.